core/src/config.rs

```diff
@@ -101,6 +101,7 @@ impl FromStr for DeviceType {
             "smartwatch" => Ok(Smartwatch),
             "chromebook" => Ok(Chromebook),
             "carthing" => Ok(CarThing),
+            "homething" => Err(()),
```

Comment:
It was fine before, the line now is redundant to the line below.

Reply:
Ah I see. No worries about the back and forth I could have been a bit more clear so I'm also partly to be blamed. Anyhow, I think the release notes, the issue and the PR here should be good enough as documentation why it was removed :)